server/ssh.go

```diff
@@ -84,6 +78,46 @@ func (me *SSHServer) authHandler(ctx ssh.Context, key ssh.PublicKey) bool {
 	return true
 }
 
+func (me *SSHServer) handleJWT(s ssh.Session) {
+	var aud []string
+	cmd := s.Command()
+	if len(cmd) > 1 {
+		aud = cmd[1:]
+	} else {
+		aud = []string{"charm"}
+	}
+	key, err := keyText(s)
+	if err != nil {
+		log.Println(err)
+		return
+	}
+	u, err := me.db.UserForKey(key, true)
+	if err != nil {
+		log.Println(err)
+		return
+	}
+	log.Printf("JWT for user %s\n", u.CharmID)
+	j, err := me.newJWT(u.CharmID, aud)
+	if err != nil {
+		log.Println(err)
+		return
+	}
+	_, _ = s.Write([]byte(j))
+	me.config.Stats.JWT()
+}
+
+func (me *SSHServer) newJWT(charmID string, audience []string) (string, error) {
```

Comment:
audience could be a `...string` to avoid typing `[]string{"item"}` 💅🏻

Reply:
Good call, just fixed this.